CHANGELOG.md

```diff
@@ -50,7 +78,11 @@ We check progress in the next reconciliation loop. ([#362](https://github.com/gi
 - Separate podsecuritypolicy.
 - Security context in deployment spec with non-root user.
 
-[Unreleased]: https://github.com/giantswarm/chart-operator/compare/v0.12.1..HEAD
+[Unreleased]: https://github.com/giantswarm/chart-operator/compare/v0.13.0..HEAD
+[v0.13.0]: https://github.com/giantswarm/chart-operator/releases/tag/v0.13.0
+[v0.12.4]: https://github.com/giantswarm/chart-operator/releases/tag/v0.12.4
+[v0.12.3]: https://github.com/giantswarm/chart-operator/releases/tag/v0.12.3
+[v0.12.2]: https://github.com/giantswarm/chart-operator/releases/tag/v0.12.2
```

Comment:
Not a blocker but for info I messed up here and used the wrong URL format. But lets fix this in master next time we need to cut a release.

Should look like.

```
[0.13.0]: https://github.com/giantswarm/chart-operator/compare/v0.12.4...v0.13.0
```

https://github.com/giantswarm/fmt/blob/master/releases/changelogs_for_managed_apps.md

Reply:
That's better to catch up on the difference. Thanks.